programs/eosio-launcher/main.cpp

```diff
@@ -466,6 +466,27 @@ struct launcher_def {
    fc::optional<uint32_t> max_transaction_cpu_usage;
    eosio::chain::genesis_state genesis_from_file;
 
+   std::string cluster_id_str() const {
+      char cluster_id_str[20];
+	   sprintf(cluster_id_str, "cluster%05u", cluster_id);
+      std::string str = cluster_id_str;
+      if (!bfs::exists(str)) {
```

Comment:
my idea is to create ```bfs::current_path() / cluster00000``` and the following files will be there:
```
shape_bridge_hosts.json
shape_bridge.json
setprods.json
bios_boot.sh
testnet.dot
last_run.json
test_keosd_out.log
test_keosd_err.log
```
There files are originally in bfs::current_path(), but not etc/eosio/. I think it's better not to put them into ```etc/eosio/cluster00000``` because the contexts of ```etc/eosio/cluster00000``` will be cleaned up after the tests and it makes user difficult to check those files.